server/lib/config/pbench-server.cfg.example

```diff
@@ -66,4 +66,4 @@ satellite-opt = /opt/pbench-server
 # The rest will come from the default config file.
 [config]
 path = %(install-dir)s/lib/config
-files = pbench-server-default.cfg
+files = pbench-put-shim.cfg, pbench-server-default.cfg
```

Comment:
I don't think we need or want references to the shim in our example config file.

Reply:
Yaa I will change that

Reply:
removed that